docs/_theme/djangodocs/layout.html

```diff
@@ -308,6 +308,7 @@
                 <div id="mc_embed_signup">
                   <div class="subscribe-form-content">
                     Stay up to date with product & security news
+                    <a target="_blank" class="newsletter-link" href="https://us13.campaign-archive.com/home/?u=9b63e92a98ecdc99732456b0e&id=f5c4f66bcf" rel="noopener">See past issues</a>
```

Comment:
Should we choose a word other than "issue". Because the title is `Stay up to date with product & security news` I immediately thought of security issues 😬 maybe "editions"?